data/bytedance

```diff
@@ -32,6 +33,8 @@ ibytedtos.com @!cn
 ibyteimg.com @!cn
 pstatp.com
 snssdk.com
+tiktokd.org
+tiktokd.net
```

Comment:
Line 36 and 37. If they are only used on tiktok, it should be included in `tiktok`

Reply:
@coderbean Hello?

Reply:
> @coderbean Hello?
Hi


> Line 36 and 37. If they are only used on tiktok, they should be moved to `tiktok`

ok, I'll do it.